database/frontend/react.json

```diff
@@ -7,6 +7,14 @@
     "subcategory": "react",
     "language": "english"
   },
+  {
+    "name": "React js",
+    "description": "Free react course - Learn fundamentals",
+    "url": "https://youtu.be/4UZrsTqkcW4",
+    "category": "frontend",
+    "subcategory": "react",
+    "language": "english"
+  },
   {
     "name": "30 days of React",
     "description": "a very nice repository and a challenge at the same time to learn ReactJS",

```

Comment:
[Nitpick] See [my feedback on this line](https://github.com/rupali-codes/LinksHub/pull/1116#discussion_r1238025458) to improve this description.